data/channels.csv

```diff
@@ -27682,6 +27685,7 @@ StarKids.tz,Star Kids,,,,TZ,,,c/TZ,swa,kids,FALSE,,,,,https://i.imgur.com/OlsUSI
 StarLife.bg,Star Life,Fox Life,,The Walt Disney Company Bulgaria EOOD,BG,,Sofia,c/BG,bul,series,FALSE,,,,https://www.starchannel-bg.com/,https://i.imgur.com/1CEnGOa.png
 StarLife.br,Star Life,Star Life Brasil,,,BR,,,c/BR,por,,FALSE,,,,https://www.star-brasil.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f9/Star_Life_2021.svg/512px-Star_Life_2021.svg.png
 StarLife.mx,Star Life,,,,MX,,,c/MX,spa,,FALSE,,,,https://www.star-latam.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f9/Star_Life_2021.svg/512px-Star_Life_2021.svg.png
+StarLife.pt,Star Life,,The Walt Disney Company Portugal,The Walt Disney Company Iberia S.L.,PT,,Lisboa,c/PT,por,lifestyle,FALSE,,,,https://www.startv.pt/programacao/starlife,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f9/Star_Life_2021.svg/800px-Star_Life_2021.svg.png

```

Comment:
Image must be under 512px.

Reply:
https://upload.wikimedia.org/wikipedia/commons/thumb/f/f9/Star_Life_2021.svg/512px-Star_Life_2021.svg.png